### PR TITLE
Update runtime and app release

### PR DESCRIPTION
**Pull request by Unrud**
This includes some minor improvements like removing whitespaces on line endings, using two spaces for indentation everywhere and setting the release date and version from git (no need to manually update the Metainfo).

### CODE REVIEW

File: freeplane.desktop.patch [hunk 1]
```diff
@@ -24,9 +24,7 @@
 +Keywords=map;outline;draft;plan;project;brainstorming;
 +Keywords[es]=mapa;esquema;borrador;plan;proyecto;tormenta de ideas;
 +
-+# This thing is to tell GnomeShell the name of the X window of the
-+# application for being able to set the correct icon, etc.
-+StartupWMClass="org-knopflerfish-framework-BundleThread", "org-knopflerfish-framework-BundleThread"
+ StartupWMClass=org-knopflerfish-framework-BundleThread
 +
 +X-Flatpak=org.freeplane.App
```
Comment (hfiguiere) [inline freeplane.desktop.patch:29]: why are you adding this?

Comment (Unrud) [reply freeplane.desktop.patch:29]: I'm not adding anything. I've only modified the patch to be compatible with the new release. The `X-Flatpak` line was already in it.

Comment (Unrud) [reply freeplane.desktop.patch]: I've now removed the `X-Flatpak` and `X-Desktop-File-Install-Version` keys. They were probably added by mistake.

**Comment by hfiguiere** [comment]
bot, build

**Comment by Unrud** [comment]
I've updated the screenshots and switched to the AppStream metadata included in the FreePlane source.

**Comment by hfiguiere** [comment]
do you need help merging this?

**Comment by hfiguiere** [comment]
bot, build

**Pull request merged**